neuralforecast/losses/pytorch.py

```diff
@@ -27,24 +28,31 @@ def _divide_no_nan(a: torch.Tensor, b: torch.Tensor) -> torch.Tensor:
     div[div == float("inf")] = 0.0
     return div
```

Comment:
If we are taking numerical protections in the denominator of certain losses, lets add the option for the numerical protection to _divide_no_nan as an alternative. See comments on relMSE.

Reply:
it wont be used because it is inside other loss. Will rewrite relmse